src/test/kotlin/com/example/realworldkotlinspringbootjdbc/presentation/ArticleControllerTest.kt

```diff
@@ -0,0 +1,237 @@
+                            }
+                        },
+                        object : ShowArticleUseCase {
+                            override fun execute(
+                                slug: String?,
+                                currentUser: Option<RegisteredUser>
+                            ): Either<ShowArticleUseCase.Error, CreatedArticle> =
+                                testCase.useCaseExecuteResult
+                        }
+                    )
+                    /**
+                     * when:
+                     */
+                    val actual = articleController.show(rawAuthorizationHeader = null, slug = pathParam)
+                    /**
+                     * then:
+                     */
+                    assertThat(actual).isEqualTo(testCase.expected)
+                }
+            }
+        }
+
+        @TestFactory
+        fun testShowWithAuthorize(): Stream<DynamicNode> {
+            return Stream.of(
+                TestCase(
+                    title = "UseCase 正常系:ユースケース（ShowArticleUseCase）が作成済記事（CreatedArticle）を返すとき、レスポンスのステータスコードが 200 になる",
+                    useCaseExecuteResult = CreatedArticle.newWithoutValidation(
+                        id = ArticleId(1),
+                        title = Title.newWithoutValidation("dummy-title"),
+                        slug = Slug.newWithoutValidation("dummy-slug"),
+                        body = ArticleBody.newWithoutValidation("dummy-body"),
+                        createdAt = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX").parse("2022-01-01T00:00:00+09:00"),
+                        updatedAt = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX").parse("2022-01-01T00:00:00+09:00"),
+                        description = Description.newWithoutValidation("dummy-description"),
+                        tagList = listOf(
+                            Tag.newWithoutValidation("dummy-tag1"),
+                            Tag.newWithoutValidation("dummy-tag2")
+                        ),
+                        authorId = UserId(1),
+                        favorited = true,
+                        favoritesCount = 1
+                    ).right(),
+                    expected = ResponseEntity(
+                        """{"article":{"title":"dummy-title","slug":"dummy-slug","body":"dummy-body","createdAt":"2021-12-31T15:00:00.000Z","updatedAt":"2021-12-31T15:00:00.000Z","description":"dummy-description","tagList":["dummy-tag1","dummy-tag2"],"authorId":1,"favorited":true,"favoritesCount":1}}""",
+                        HttpStatus.valueOf(200)
+                    )
+                ),
+                TestCase(
+                    title = "UseCase 準正常系:ユースケース（ShowArticleUseCase）が NotFound エラー（NotFound）を返すとき、レスポンスのステータスコードが 404 になる",
+                    useCaseExecuteResult = ShowArticleUseCase.Error.NotFoundArticleBySlug(object : MyError {}, Slug.newWithoutValidation("dummy-slug")).left(),
```

Comment:
[nits]
すいませんここ名前付き引数にしてもらっても良いでしょうか？ 🙇 

```suggestion
                    useCaseExecuteResult = ShowArticleUseCase.Error.NotFoundArticleBySlug(
                        〇〇 = object : MyError {},
                        〇〇 = Slug.newWithoutValidation("dummy-slug")
                    ).left(),
```

Reply:
こちら修正しました。

https://github.com/sunakan/realworld-kotlin-springboot-jdbc/pull/97/commits/5df847009f72da75f793d069c4a58aa18bad4090